src/Lesson3/Task3.java

```diff
@@ -0,0 +1,19 @@
+package Lesson3;
+
+public class Task3 {
+    public static void main(String[] args) {
+        String word1 = "The End";
+        String word2 = "End The";
+        String check = "The";
+        if (word1.startsWith(check) == true) {
+            System.out.println("word1 starts with " + check);
+        } else  {
+            System.out.println("word1 doesn't starts with " + check);
+        }
+        if (word2.startsWith(check) == true) {
+            System.out.println("word2 starts with " + check);
+        } else  {
+            System.out.println("word2 doesn't starts with " + check);
+        }
```

Comment:
You need just to print true or false if string starts with 'The'. So, code should be much shorter.